Core new json format fix

Alex-Choi99 opened the pull request.

Alex-Choi99 committed:
Now MapManager.cs can read lower_mainland_clamped_height.json

Alex-Choi99 committed:
Merge branch 'core' into core_new_json_format_fix

# Conflicts:
#	Assets/Textures/UI.meta
#	Assets/Textures/UI/TTT_UI_Energy.svg.meta
#	Assets/Textures/UI/TTT_UI_Industry.svg.meta
#	Assets/Textures/UI/TTT_UI_Nature.svg.meta
#	Assets/Textures/UI/TTT_UI_Renewable.svg.meta
#	Assets/Textures/UI/TTT_UI_Residential.svg.meta
#	Assets/Textures/UI/Workfiles.meta
#	Assets/Textures/UI/Workfiles/TTT_UI_Energy.af.meta
#	Assets/Textures/UI/Workfiles/TTT_UI_Industry.af.meta
#	Assets/Textures/UI/Workfiles/TTT_UI_Nature.af.meta
#	Assets/Textures/UI/Workfiles/TTT_UI_Renewable.af.meta
#	Assets/Textures/UI/Workfiles/TTT_UI_Residential.af.meta

Donut11one committed:
file browser addons

Donut11one committed:
Merge branch 'core' of https://github.com/TP-3T/4957-Tides into core

Donut11one committed:
Merge branch 'core' of https://github.com/TP-3T/4957-Tides into core

Donut11one committed:
Small meta files

Donut11one committed:
Deleted files from core so i can merge in

These files are suppose to be deleted im deleting them so this branch can be merged with core

Donut11one committed:
Merge branch 'core' into core_new_json_format_fix

Donut11one commented:
Working on merging core with this branch since other prs were made I will approve and merge this branch

Donut11one committed:
Fixing LFS file tracking for wrongly committed binaries

Donut11one committed:
Fixed Hex bug

Now when you click the hex it gives you the proper hex instead of clamping it to the row

Donut11one reviewed (approved):
Thanks for updating the json format Didnt check for bugs but i got them fixed 👍
